eip712/messages.py

```diff
@@ -140,7 +141,11 @@ def __getitem__(self, key: str) -> Any:
     def signable_message(self) -> SignableMessage:
         """The current message as a :class:`SignableMessage` named tuple instance."""
         return SignableMessage(
-            HexBytes(b"\x01"),
+            HexBytes(bytes.fromhex("1901")),
             HexBytes(hash_domain(self._domain_)),
             HexBytes(hash_eip712_message(self._body_)),
         )
+
+    @property
+    def message_hash(self) -> HexBytes:
```

Comment:
@fubuloubu I fixed the magic number and just added a new property on the object itself, does this work? 
turns out i just didnt know what i was doing before